pkg/protocols_test.go

```diff
@@ -59,19 +59,30 @@ func TestHttpProbe(t *testing.T) {
 // Creates an HTTP server for testing.
 func newTestHTTPServer(t *testing.T) *http.Server {
 	hostPort := net.JoinHostPort("127.0.0.1", "8080")
-	server := &http.Server{Addr: hostPort}
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+	mux := http.NewServeMux()
```

Comment:
We don't need a mux for this simple scenario. The line 63 should be enough.

Reply:
okay got it

Reply:
Thank you :)